.github/workflows/ci.yml

```diff
@@ -1,6 +1,7 @@
 name: ci
 
 on:
+  push:
```

Comment:
Forgot that we need this to make the CI run on any push to the branch, otherwise, it only runs on the initial PR open.